server/src-lib/Hasura/Server/Logging.hs

```diff
@@ -118,118 +117,133 @@ instance ToJSON WebHookLog where
 
 type WebHookLogger = WebHookLog -> IO ()
 
-data AccessLog
-  = AccessLog
-  { alStatus       :: !N.Status
-  , alMethod       :: !T.Text
-  , alSource       :: !T.Text
-  , alPath         :: !T.Text
-  , alHttpVersion  :: !N.HttpVersion
-  , alDetail       :: !(Maybe Value)
-  , alRequestId    :: !(Maybe T.Text)
-  , alHasuraUser   :: !(Maybe UserVars)
-  , alQueryHash    :: !(Maybe T.Text)
-  , alResponseSize :: !(Maybe Int64)
-  , alResponseTime :: !(Maybe Double)
+-- | Log information about the HTTP request
+data HttpInfoLog
+  = HttpInfoLog
+  { hlStatus      :: !N.Status
+  , hlMethod      :: !T.Text
+  , hlSource      :: !T.Text
+  , hlPath        :: !T.Text
+  , hlHttpVersion :: !N.HttpVersion
   } deriving (Show, Eq)
 
-instance L.ToEngineLog AccessLog where
-  toEngineLog accessLog =
-    (L.LevelInfo, "http-log", toJSON accessLog)
-
-instance ToJSON AccessLog where
-  toJSON (AccessLog st met src path hv det reqId hUser qh rs rt) =
+instance ToJSON HttpInfoLog where
+  toJSON (HttpInfoLog st met src path hv) =
     object [ "status" .= N.statusCode st
            , "method" .= met
            , "ip" .= src
            , "url" .= path
            , "http_version" .= show hv
-           , "detail" .= det
-           , "request_id" .= reqId
-           , "user" .= hUser
-           , "query_hash" .= qh
-           , "response_size" .= rs
-           , "query_execution_time" .= rt
            ]
 
-data LogDetail
-  = LogDetail
-  { _ldQuery :: !TBS.TByteString
-  , _ldError :: !Value
+-- | Information about a GraphQL operation over HTTP
```

Comment:
Can `OperationLog` is used for `/v1/query`?

If yes then this comment is not correct.